executor/builder.go

```diff
@@ -1145,6 +1152,7 @@ func buildNoRangeTableReader(b *executorBuilder, v *plan.PhysicalTableReader) (*
 		desc:         ts.Desc,
 		columns:      ts.Columns,
 		priority:     b.priority,
+		feedback:     statistics.NewQueryFeedback(ts.Table.ID, pkID, ts.HistVersion, ts.StatsInfo().Count()),
```

Comment:
The pkID may conflict with index ID?

Reply:
Yes, but we can infer them from the ranges stored in the feedback.

Reply:
I'll merge IndexRange with IntColumnRange soon...